libraries/AP_DDS/AP_DDS_Client.cpp

```diff
@@ -42,6 +43,7 @@ static constexpr uint16_t DELAY_GEO_POSE_TOPIC_MS = 33;
 static constexpr uint16_t DELAY_CLOCK_TOPIC_MS = 10;
 static constexpr uint16_t DELAY_GPS_GLOBAL_ORIGIN_TOPIC_MS = 1000;
 static constexpr uint16_t DELAY_PING_MS = 500;
+static constexpr uint16_t DELAY_STATUS_TOPIC_MS = 500;
```

Comment:
Can we make this event driven instead rather than poll?

Reply:
Are there events I can use to drive this?

Reply:
There are not. ArduPilot does not have any such architecture. The best approach is to poll the state quickly and only publish on change.

Reply:
set to 100ms